Dockerfile

```diff
@@ -80,7 +80,10 @@ RUN python3 -m pip install --no-deps \
         pytijo==0.0.2 \
         pyzbar-upright==0.1.8 \
         rawpy==0.19.1 \
-        scipy==1.11.4
+        scipy==1.11.4 \
```

Comment:
Please also add it here:
https://github.com/microsoft/lamar-benchmark/blob/main/pyproject.toml#L36

This is for the local installation.